spot_driver_cpp/include/spot_driver_cpp/api/default_image_api.hpp

```diff
@@ -14,13 +16,10 @@ namespace spot_ros2 {
 /**
  * @brief Implements SpotInterfaceBase to use the Spot C++ SDK.
  */
-class SpotInterface : public SpotInterfaceBase {
+class DefaultImageApi : public ImageApi {
```

Comment:
I don't think this is an accurate use of the term "API" --  the library as a whole _provides_ an API. I think it's more correct to refer to the base class here as something like `ImageClientInterface`, since it's a pure virtual class, and then the child class can be `DefaultImageClient`.